km61/Mozghovyi_Mykyta/laboratory1.sql

```diff
@@ -1,2 +1,91 @@
 -- LABORATORY WORK 1
 -- BY Mozghovyi_Mykyta
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+створювати таблиці
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER student
+IDENTIFIED BY P123
+TEMPORARY TABLESPACE "USER";
+GRANT CREATE TABLE TO student;
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Студент має мобільний номер іноземного оператора.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE TABLE student(
+ student_name varchar(20) NOT NULL
```

Comment:
`student_phone number(20, 0) NOT NULL,`
`student_surname varchar(20) NOT NULL,`
`student_group varchar(20) NOT NULL`